analytics_data_api/v0/tests/views/test_course_summaries.py

```diff
@@ -162,3 +162,11 @@ def test_no_matching_courses(self):
     def test_bad_course_id(self, course_ids):
         response = self.authenticated_get(self.path(course_ids=course_ids))
         self.verify_bad_course_id(response)
+
+    def test_collapse_upcoming(self):
+        self.generate_data(availability='Starting Soon')
```

Comment:
I think it would be helpful to generate data for "Upcoming" too and ensure that the result is the sum of the two.